### ISSUE TITLE
Dry run (feature request)

### ISSUE THREAD

**Issue by everlof** [venmo/synx#120]
I love the feature of various git commands, '--dry-run'.

I'd love to have the same argument here, to simply do a dry run. Print structure, files, problems etc, but don't do the actual moving.

If I knew any ruby I'd try, but sadly I don't.

**Comment by ecnepsnai** [comment]
Agreed, would be super useful with CI if the exit code was 0 for no changes, and 1 if any changes would happen.

**Comment by gtatgh** [comment]
> Agreed, would be super useful with CI if the exit code was 0 for no changes, and 1 if any changes would happen.

Exactly my opinion.
